lib/locale/pl_PL/strings.php

```diff
@@ -2564,6 +2564,11 @@
 $_LANG['- all states -'] = '- wszystkie -';
 $_LANG['Country state:'] = 'Województwo:';
 $_LANG['customers to groups assignment'] = 'przypisywanie klientów do grup';
+$_LANG['Assigned Customer'] = 'Przypisany klient';
+$_LANG['VLAN List'] = 'Lista VLAN';
+$_LANG['VLAN Description'] = 'Opis VLANu';
```

Comment:
Tu pytanie do @awbnet: VLANu to poprawnie? :)

Reply:
Zostałem grammatic whore projektu :D

Trzeba to zamienić na:
```
$_LANG['VLAN List'] = 'Lista VLAN-ów';
$_LANG['VLAN Description'] = 'Opis VLAN-u';
```
Tzw. dywiz (łacznik) czyli "-" (w tym przypadku nie chodzi o myślnik), stosujemy przy odmianie właśnie skrótów: SMS, LAN, VLAN, WAN, itp.

Apostrof dajemy, kiedy wyraz ma ostatnią literę, której się nie wymawia tj. np. "pendrive". Nie wymawiamy "e" na końcu, dlatego przy odmianie "pendrive'a", zaznaczamy apostrofem, że "e" jest niewymawialne.